modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -59,4 +59,58 @@ test.describe('Manage object fields through Model Builder', () => {
 			ListTypeDefinition.id
 		);
 	});
+
+	test('can delete object field', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		await page.goto('/');
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
```

Comment:
Hey @rafaellajordao, we have a subtle difference in `postRandomObjectDefinition` and `openObjectFolder` functions. In the first function the object definitions needs to refer the object folder external reference code `'default'`. But in the openObjectFolder, we need the name of the object folder to use in URL  `'Default'`. This is a small change, but this can do the difference in the end.